python/inline/conftest.py

```diff
@@ -470,13 +495,24 @@ def parse_constructor(self, node):
                     # check if "disabled" is a boolean
                     elif (
                         index == 4
-                        and isinstance(arg, ast.NameConstant)
-                        and isinstance(arg.value, bool)
+                        and isinstance(arg, ast.Num)
+                        and isinstance(arg.value, int)
```

Comment:
Can you recover these two lines? They are about the feature "disabled", which is boolean

Reply:
resolved